### DISCUSSION TITLE
Message: Multistring must contain at least one string

### DISCUSSION THREAD

**Discussion by ForceOptix** [WeblateOrg/weblate#10864] [Q&A]
Hello at all,
I currently evaluate weblate as tool for our translation topic and have used our already existing translation file created with BabelEdit in I18n JSON style for Angular. I used this data as basefile but always get the error message "multistring must contain at least one string". I don't understand what I have to search for to fix this file. Can someone explain this to me?
Selected file format in weblate: JSON go-I18n-v2
![Monosnap basefile — weblate-test 2024-01-26 10-50-31](https://github.com/WeblateOrg/weblate/assets/14006351/70b9ed32-31fe-45bf-94f4-bb57d093f02b)

**Comment by nijel** [comment]
Looks like translate-toolkit is having issues when handling plurals in your file.

* Can you share your file so that we can take a look?
* Maybe go-I18n-v2 is not the right format for you. If you are not using plurals, choose simple "JSON nested structure file".

**Reply by ForceOptix** [reply]
You are right. JSON nested structure file fixed my issue.
Thx for the fast response.

**Reply by nijel** [reply]
After https://github.com/translate/translate/pull/5175 it should produce a better error message.
